python/dgl/nn/pytorch/conv/gatedgcnconv.py

```diff
@@ -0,0 +1,180 @@
+"""Torch Module for GatedGCN layer"""
+# pylint: disable= no-member, arguments-differ, invalid-name, cell-var-from-loop
+import torch
+import torch.nn.functional as F
+from torch import nn
+
+from .... import function as fn
+
+
+class GatedGCNConv(nn.Module):
+    r"""GatedGCN: Residual Gated Graph ConvNets
+        <https://arxiv.org/pdf/1711.07553v2.pdf>`
+
+    .. math::
+        e_{ij}^{l+1}=D^{l} h_{i}^{l}+E^{l} h_{j}^{l}+C^l e_{ij}^{l}
+
+        norm_{ij}=\Sigma_{j\in N_{i}} \sigma\left(e_{ij}^{l+1}\right)+\varepsilon
+
+        \hat{e}_{ij}^{l+1}=\sigma(e_{ij}^{l+1}) \div norm_{ij}
+
+        h_{i}^{l+1}=A^l h_{i}^{l}+\Sigma_{j \in N_{i}} \hat{e}_{ij}^{l+1} \odot B^l h_{j}^{l}
+
+    Parameters
+    ----------
+    in_feats : int
+        Input feature size; i.e, the number of dimensions of :math:`\mathbf{x}_i`.
+    edge_feats: int
+        Edge feature size; i.e., the number of dimensions of :math:\mathbf{e}_{j,i}`.
+    out_feats : int
+        Output feature size; i.e., the number of dimensions of :math:`h_i^{(t+1)}`.
+    dropout : float, optional
+        Dropout rate on node and edge feature. Defaults: ``0``.
```

Comment:
```suggestion
        Dropout rate on node and edge feature. Default: ``0``.
```
for consistency with the cases below